antenna/breakpad_resource.py

```diff
@@ -368,45 +393,76 @@ def on_post(self, req, resp):
             resp.body = 'CrashID=%s%s\n' % (self.config('dump_id_prefix'), crash_id)
 
     def hb_run_crashmover(self):
-        """Check to see if it should spawn a crashmover and does if appropriate."""
+        """Spawn a crashmover if there's work to do."""
         # Spawn a new crashmover if there's stuff in the queue and there isn't
         # one currently running
         if self.crashmover_save_queue and self.crashmover_pool.free_count() > 0:
```

Comment:
Oops--this needs to factor in the `self.crashmover_publish_queue`, too.

Mmm... part of me wants to generalize the queue thing now.